Firmware/eeprom.h

```diff
@@ -37,313 +37,321 @@ typedef struct
+| 0x0D11 3345 | float   | EEPROM_UVLO_ACCELL                    | ???          | ff ff ff ffh          | Power panic saved normal acceleration             | ???          | D3 Ax0d11 C4
+| 0x0D0D 3341 | float   | EEPROM_UVLO_RETRACT_ACCELL            | ???          | ff ff ff ffh          | Power panic saved retract acceleration            | ???          | D3 Ax0d0d C4
+| 0x0D09 3337 | float   | EEPROM_UVLO_TRAVEL_ACCELL             | ???          | ff ff ff ffh          | Power panic saved travel acceleration             | ???          | D3 Ax0d09 C4
+| 0x0D05 3333 | unint32 | EEPROM_JOB_ID                         | ???          | 00 00 00 00h            | Job ID used by host software                    | D3 only      | D3 Ax0d05 C4
+| 0x0D04 3332 | uint8   | EEPROM_ECOOL_ENABLE                   | ffh 255      | ^                     | Disable extruder motor scaling for non-farm print | LCD menu     | D3 Ax0d04 C1
+| ^           | ^       | ^                                     | 2ah 42       | ^                     | Enable extruder motor scaling for non-farm print  | ^            | D3 Ax0d04 C1
+| 0x0D03 3331 | uint8   | EEPROM_FW_CRASH_FLAG                  | ffh 255      | ffh 255               | Last FW crash reason (dump_crash_reason)          | D21/D22      | D3 Ax0d03 C1
+| ^           | ^       | ^                                     | 00h 0        | ^                     | manual                                            | ^            | ^
+| ^           | ^       | ^                                     | 01h 1        | ^                     | stack_error                                       | ^            | ^
+| ^           | ^       | ^                                     | 02h 2        | ^                     | watchdog                                          | ^            | ^
+| ^           | ^       | ^                                     | 03h 3        | ^                     | bad_isr                                           | ^            | ^
+| ^           | ^       | ^                                     | 04h 4        | ^                     | bad_pullup_temp_isr                               | ^            | ^
+| ^           | ^       | ^                                     | 05h 5        | ^                     | bad_pullup_step_isr                               | ^            | ^
+| 0x0D02 3330 | uint8   | EEPROM_TEMP_MODEL_ENABLE              | 00h 0        | ff/00                 | Temp model deactivated                            | Temp model   | D3 Ax0d02 C1
+| ^           | ^       | ^                                     | 01h 1        | ^                     | Temp model activated                              | ^            | ^
+| 0X0CFE 3326 | float   | EEPROM_TEMP_MODEL_P                   | ???          | ff ff ff ffh          | Temp model power (W)                              | Temp model   | D3 Ax0cfe C4
```

Comment:
Addresses/values/descriptions are good. Would be nice to use lower-case x for the hex address though. (0X0 -> 0x0)

Reply:
Fixed https://github.com/3d-gussner/Prusa-Firmware/commit/bab83704a1c4b0931db80df7f148203324d0e2ed